src/Symfony/Component/Security/Http/Authenticator/Passport/Credentials/AccessTokenCredentials.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Authenticator\Passport\Credentials;
+
+/**
+ * Implements token credentials.
+ *
+ * The token is checked by the TokenHandler during
+ * authentication.
+ *
+ * @author Florent Morselli <florent.morselli@spomky-labs.com>
+ *
+ * @final
+ */
+class AccessTokenCredentials implements CredentialsInterface
+{
+    public function __construct(private readonly string $token)
+    {
+    }
+
+    public function getToken(): string
```

Comment:
```suggestion
    public function getAccessToken(): string
```

Just a suggestion, I'm not sure if it's redundant or useful to explicitly differentiate between "access token" and authentication tokens (the one created after the passport checking).

Reply:
👍 consistent with `AccessTokenExtractorInterface::extractAccessToken()` and useful to me